app/bundles/FormBundle/Resources/views/Builder/index.html.twig

```diff
@@ -194,7 +194,7 @@
               {{ form_row(form.noIndex) }}
               {{ form_row(form.inKioskMode) }}
               {{ form_row(form.renderStyle) }}
-              {{ form_row(form.template) }}
+              <div id="theme-selector">{{ form_row(form.template) }}</div>
```

Comment:
Please add the ID in the form definition itself: https://github.com/mautic/mautic/blob/5.x/app/bundles/FormBundle/Form/Type/FormType.php#L92

Reply:
Done!